<?xml version="1.0"?>
<launch>
    <!-- <include file="$(find base_control)/launch/base_control.launch"/> -->
	<arg name="base_frame" default="/base_footprint"/>
    <arg name="map_frame" default="/map"/>

    <node name="laser2base" pkg="tf" type="static_transform_publisher" 
            args="0 0 0 0 0 0 /map /laser 20"/>  
    <node name="map" pkg="tf" type="static_transform_publisher"
            args="0 0 0 0 0 0 /map /odom 20"/>

    <!-- <node name="scan2pointcloud" pkg="robot_perception" type="scan2pointcloud" output="screen">
         <param name="base_frame" type="string" value="(arg base_frame)"/>
    </node>  -->

    <!-- <node name="test" pkg="robot_perception" type="Frame" output="screen">
        <param name="base_frame" value="/laser"/>
    </node> -->

	<!-- <node name="percept2" pkg="robot_perception" type="division" output="screen">
		<param name="base_frame" value="${arg base_frame}"/>
	</node> -->

    <node name="scan2" pkg="robot_perception" type="scan2" output="screen">
        <param name="map_frame" value="/map"/>
    
    </node>
    <!-- <node name="perception1" pkg="robot_perception" type="perception2d" output="screen">
        <param name="map_frame" value="/map"/>
    </node> -->

	<!--<node name="rviz" pkg="rviz" type="rviz" args="-d $(find robot_perception)/rviz/point.rviz"/>-->
    <!-- <node name="rviz" pkg="rviz" type="rviz" />  -->
</launch> 
